.github/workflows/ci-aarch64.yml

```
@@ -0,0 +1,146 @@
+#===============================================================================
+# Copyright contributors to the oneDAL project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+name: "CI AArch64"
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+env:
+  DEP_DIR: ${{ github.workspace }}/__deps
+
+# Stop stale workflows when pull requests are updated: https://stackoverflow.com/a/70972844
+# Does not apply to the main branch.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  build-and-test:
+    strategy:
+      matrix:
+        os: [
+          { name: Arm/GitHub hosted, label: ubuntu-24.04-arm },
+          { name: Arm/Arm hosted, label: ah-ubuntu_24_04-c6g_2x-50 }
```

I guess my previous comment about naming was ambiguous, AArch64 will be sufficient.
```suggestion
          { name: GitHub hosted, label: ubuntu-24.04-arm },
          { name: Arm hosted, label: ah-ubuntu_24_04-c6g_2x-50 }
```

---

Done.